airflow/providers/databricks/operators/databricks.py

```diff
@@ -944,6 +952,7 @@ def __init__(
         databricks_retry_args: dict[Any, Any] | None = None,
         wait_for_termination: bool = True,
         databricks_conn_id: str = "databricks_default",
+        databricks_workflow_run_metadata: dict | None = None,
```

Comment:
do we need the `_databricks` prefix?

Reply:
yes, thanks, that's not needed as such. I have removed the prefix and kept it as `workflow_run_metadata`